recipe/meta.yaml

```diff
@@ -154,7 +154,7 @@ outputs:
     script_interpreter: >-
       python -m vc_repack --activate --arch {{ target_platform }} --version {{ runtime_version }}
       --activate-year {{ vsyear }} --activate-major {{ vsver }} --activate-vcver {{ vcver }}
-      --activate-vcvars-ver {{ cl_version.split(".")[0]|int - 5 }}.{{ cl_version.split(".")[1] }}
+      --activate-vcvars-ver {{ runtime_version.split(".")[0] }}.{{ runtime_version.split(".")[1] }}
```

Comment:
This is wrong. Why did you make this change?

Reply:
Because I was trying to fix the not-found version at the time, and looking for 14.33 when we're on 14.32 seemed like a potential issue. Happy to rebase that out.